applications/RANSModellingApplication/python_scripts/turbulence_model_factory.py

```diff
@@ -0,0 +1,25 @@
+import KratosMultiphysics
+import KratosMultiphysics.RANSModellingApplication
+
+from KratosMultiphysics.RANSModellingApplication.evm_k_epsilon_configuration import TurbulenceKEpsilonConfiguration
+
+def Factory(settings, Model, parallel_type = "OpenMP"):
+    if (type(settings) != KratosMultiphysics.Parameters):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string"
+        )
+    if (type(Model) != KratosMultiphysics.Model):
+        raise Exception("expected input shall be a Model object")
+    turbulence_models_list = ["k_epsilon"]
+
+    model_type = settings["model_type"].GetString()
+    if not model_type in turbulence_models_list:
+        msg = "Uknown turbulence \"model_type\" name : \"" + model_type
+        msg += "\".\nSupported \"model_type\" names: " + turbulence_models_list.__str__(
```

Comment:
```suggestion
        msg += "\".\nSupported \"model_type\" names: " + str(turbulence_models_list
```
Prefer using standard python functions instead of "hidden" methods.